iron-a11y-keys-behavior.html

```diff
@@ -411,14 +374,18 @@
           event.stopPropagation();
         }
 
-        keyBindings.forEach(function(keyBinding) {
-          var keyCombo = keyBinding[0];
-          var handlerName = keyBinding[1];
+        if (event.defaultPrevented) {
```

Comment:
Maybe this check should be inside the for loop, in case one of the triggered listeners prevents the original event